frontend/src/components/common/Header.svelte

```diff
@@ -82,8 +75,14 @@
         </div>
       </div>
       {#if user.isAuthorized}
+        <div class="ml-auto">
+          <LanguageSwitcher />
+        </div>
         <Profile.Desktop items={userMenuItems} />
       {:else}
+        <div class="ml-auto">
+          <LanguageSwitcher />
+        </div>
```

Comment:
If user authorized you render Language Switcher and if user not you also render it? (Explain logic if I am wrong)